libhb/hb.c

```diff
@@ -2297,10 +2297,12 @@ static void redirect_thread_func(void * _data)
     if (pipe(pfd))
        return;
 #if defined( SYS_MINGW )
-    // dup2 doesn't work on windows for some stupid reason
-    stderr->_file = pfd[1];
+    // Non-console windows apps do not have a stderr->_file
+    // assigned properly
+    (void) freopen("NUL", "w", stderr);
```

Comment:
Question, does this leave a file named "NUL" somewhere that needs to be cleaned up?

Reply:
It should not. `NUL` is the old DOS device which Windows will convert behind the scenes to `\Device\Null`, its `/dev/null` equivalent.

![windows-nul](https://github.com/HandBrake/HandBrake/assets/12900645/77da0ef1-0cf6-47e7-affd-ca350e7ce09c)